std/typecons.d

```diff
@@ -1174,11 +1174,19 @@ if (distinctFieldNames!(Specs))
             Returns:
                 A `size_t` representing the hash of this `Tuple`.
          */
-        size_t toHash() const nothrow @trusted
+        size_t toHash() const nothrow @safe
```

Comment:
Can this also be `pure`? I believe it should be.

Reply:
It can't because the base signature of `TypeInfo.getHash` doesn't include `pure`.